manifests/kustomize/env/aws/README.md

```diff
@@ -0,0 +1,82 @@
+# Sample installation
```

Comment:
Follow boggy's suggestion, add a OWNERs file under `manifests/kustomize/env/aws` you can put you and me as approver

Reply:
Added `OWNERS` file